browser/components/newtab/tests/xpcshell/test_AboutNewTabService.js

```diff
@@ -123,7 +126,17 @@ add_task(function* test_updates() {
   notificationPromise = nextChangeNotificationPromise(
     DEFAULT_HREF, "Remote href changes back to default");
   Preferences.set("general.useragent.locale", "en-US");
+  yield notificationPromise;
+
+  expectedPath = expectedPath.replace("/en-GB/", "/en-US/");
```

Comment:
Can you please add a comment above this block describing you are testing mode changing?